workers/unity/Packages/com.improbable.gdk.core/Worker/Worker.cs

```diff
@@ -9,19 +9,46 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     Represents a SpatialOS worker.
+    /// </summary>
     public class Worker : IDisposable
     {
+        /// <summary>
+        ///     The origin of the worker in local Unity space.
+        /// </summary>
         public readonly Vector3 Origin;
+
+        /// <summary>
+        ///     The type of the worker.
+        /// </summary>
         public readonly string WorkerType;
+
+        /// <summary>
+        ///     The worker ID.
```

Comment:
should we add that it is unique

Reply:
Have done so (noting its only unique for a SpatialOS deployment)